src/main/webapp/tests/Components/FileInputComponent.test.jsx

```diff
@@ -180,7 +187,9 @@ test('Check that the proper error message is displayed for invalid file size.',
     await userEvent.upload(fileInput, invalidFile);
 
     // Check that file is present
-    expect(screen.getByTestId('label')).toHaveTextContent('test.json');
+    await waitFor(() => {
+        expect(screen.getByTestId('label')).toHaveTextContent('test.json');
+    });
```

Comment:
also a minor one, wdyt about using `await screen.findByTestId ` instead of `waitFor + getByTestId` should be a bit cleaner but basically doing the same job, the only difference is that `find` will be rejected when it will find many fitting results.

Reply:
That's a good catch!

> The findBy and findAllBy queries are async and retry until the query returns successfully, or when the query times out; they wrap waitForElement

Exactly what we need